### ISSUE TITLE
Properties `text-underline-offset` and `text-decoration-thickness` false-positives

### ISSUE THREAD

**Issue by WaKeMaTTa** [sds/scss-lint#997]
The linter is raising this issue:

```
app/assets/stylesheets/components/_link.scss:8:3 [W] PropertySpelling: Unknown property text-underline-offset
app/assets/stylesheets/components/_link.scss:9:3 [W] PropertySpelling: Unknown property text-decoration-thickness
```

When `text-underline-offset` and `text-decoration-thickness` are real properties:

- https://developer.mozilla.org/en-US/docs/Web/CSS/text-underline-offset
- https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-thickness
